internal/test/e2e/cluster.go

```diff
@@ -830,43 +836,56 @@ func (c *TestCluster) waitForHostContracts(hosts map[types.PublicKey]struct{}) {
 	})
 }
 
-// testNetwork returns a custom network for testing which matches the
-// configuration of siad consensus in testing.
-func testNetwork() *consensus.Network {
-	n := &consensus.Network{
-		InitialCoinbase: types.Siacoins(300000),
-		MinimumCoinbase: types.Siacoins(299990),
-		InitialTarget:   types.BlockID{4: 32},
+func (c *TestCluster) mineBlocks(addr types.Address, n uint64) error {
+	for i := uint64(0); i < n; i++ {
+		if block, found := coreutils.MineBlock(c.cm, addr, time.Second); !found {
+			return errors.New("failed to find block")
+		} else if err := c.Bus.AcceptBlock(context.Background(), block); err != nil {
+			return err
+		}
 	}
+	return nil
+}
 
-	n.HardforkDevAddr.Height = 3
-	n.HardforkDevAddr.OldAddress = types.Address{}
-	n.HardforkDevAddr.NewAddress = types.Address{}
-
-	n.HardforkTax.Height = 10
-
-	n.HardforkStorageProof.Height = 10
-
-	n.HardforkOak.Height = 20
-	n.HardforkOak.FixHeight = 23
-	n.HardforkOak.GenesisTimestamp = time.Now().Add(-1e6 * time.Second)
+func convertToCore(siad encoding.SiaMarshaler, core types.DecoderFrom) {
```

Comment:
We shouldn't need this anymore. Probably fine to remove in a f/u.

There is also an unused `convertToCore` in `stores/wallet.go`. The other ones probably need to wait for `hostd` to finish migrating.

Reply:
I'll remove it in a F/U. I'm still using siad's genesis block.